fud/fud/stages/vivado/extract.py

```diff
@@ -69,6 +71,26 @@ def futil_extract(directory):
         f8_muxes = to_int(find_row(slice_logic, "Site Type", "F8 Muxes")["Used"])
         f9_muxes = to_int(find_row(slice_logic, "Site Type", "F9 Muxes")["Used"])
 
+        # Insert info into resourceInfo
+        resourceInfo.update({
+            "lut": to_int(find_row(slice_logic, "Site Type", "CLB LUTs")["Used"]),
+            "dsp": to_int(find_row(dsp_table, "Site Type", "DSPs")["Used"]),
+            "meet_timing": int(meet_timing),
+            "registers": rtl_component_extract(directory, "Registers"),
+            "muxes": rtl_component_extract(directory, "Muxes"),
+            "clb_registers": clb_reg,
+            "carry8": carry8,
+            "f7_muxes": f7_muxes,
+            "f8_muxes": f8_muxes,
+            "f9_muxes": f9_muxes,
+            "clb": clb_lut + clb_reg + carry8 + f7_muxes + f8_muxes + f9_muxes,
+        })
+
+    except Exception as e:
+        traceback.print_exc(e)
+        print("Implementation files weren't found, skipping.", file=sys.stderr)
```

Comment:
I know this was a problem in the code already, but it might be nicer to narrow the exception handler to just the file-reading code that might fail, and to use `OSError` instead of a blanket `Exception` (which can cause unexpected behavior when other things go wrong, like typos in variable names).

Reply:
(Unless I'm misunderstanding and wider error handling is intentional because stuff like `find_row` might fail…)

Reply:
Considering this resolves since it got fixed in master.